src/main/java/com/Novi/TechItEasy/services/CI_ModuleService.java

```diff
@@ -0,0 +1,116 @@
+package com.Novi.TechItEasy.services;
+
+import com.Novi.TechItEasy.dtos.CI_ModuleDto;
+import com.Novi.TechItEasy.dtos.CI_ModuleInputDto;
+import com.Novi.TechItEasy.exceptions.RecordNotFoundException;
+import com.Novi.TechItEasy.helpers.DtoConverters;
+import com.Novi.TechItEasy.models.CI_Module;
+import com.Novi.TechItEasy.repositories.CI_ModuleRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class CI_ModuleService {
+
+    private final CI_ModuleRepository ci_moduleRepository;
+
+    public CI_ModuleService(CI_ModuleRepository ci_moduleRepository) {
+        this.ci_moduleRepository = ci_moduleRepository;
+    }
+
+
+    /// For fetching all CI modules currently in the database /////
+    public List<CI_ModuleDto> getCiModules() {
+
+        List<CI_Module> CI_Modules = new ArrayList<>(ci_moduleRepository.findAll());
```

Comment:
Het `= new Arraylist` gedeelte is slim bedacht, maar niet nodig, omdat de "findAll()" methode van JpaRepository al een List<t> returned. Had je een CrudRepository extend in je CiModuleRepository, dan was dit wel handig geweest, omdat de "findAll()" methode daar een Collection returned en dus geen List